### DISCUSSION TITLE
Stop livestream download when an ad comes up

### DISCUSSION THREAD

**Discussion by kvicken** [streamlink/streamlink#5091] [Q&A] [answered]
I noticed that when twitch start playing ads, streamlink stops downloading the livestream and resumes right after the ads are over. I want to know if it is possible to stop and resume downloading the livestream when an ad starts playing?

What i basically want is to have a video file prior to the ads and a new one after the ads.

**Comment by bastimeyer** [comment, marked as answer]
Out of scope, see #4973

**Reply by kvicken** [reply]
Okay, thanks for answering.
